File: Is Subsequence/Is_Subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        if (s.length() == 0){
            return true;
        }
        int a = 0, b = 0;
        while (b < t.length()){
            if (t.charAt(b) == s.charAt(a)){
                a++;
                if (a == s.length()){
                    return true;
                }
            }
            b++;
        }
        return false;
    }
}

File: Coding Problems & My Solutions/Is Subsequence/Is_Subsequence.swift
class Solution {
    func isSubsequence(_ s: String, _ t: String) -> Bool {
        if s.isEmpty {
            return true
        }
        var b = 0
        var sArr = Array(s)
        var tArr = Array(t)
        for char in tArr {
            if char == sArr[b] {
                b += 1
                if b >= sArr.count {
                    return true
                }
            }
        }
        return false
    }
}